parser/parser_test.go

```diff
@@ -1,15 +0,0 @@
-package parser
-
-import (
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-)
-
-func Test_ParseString(t *testing.T) {
-	assert := assert.New(t)
-	input := "A little more than 24 hours from now Trump will declare victory.\n\nIf Republicans do well, Trump will take credit\n\nIf Democrats do well Trump will say it would have been much worse without his rallies. He will also blame “illegal” voters and the media.\n\nBookmark this."
-	output := PrintAndParse(input)
-	expected := []string{"little", "hours", "trump", "declare", "victory", "republicans", "well", "trump", "take", "credit", "democrats", "well", "trump", "say", "would", "much", "worse", "without", "rallies", "also", "blame", "illegal", "voters", "media", "bookmark"}
-	assert.Equal(expected, output)
```

Comment:
why is this file deleted if it still able to run and pass we should leave it to maintain the functionality of the underlying logic

Reply:
It wasn't deleted. It was simply moved to PrintAndParse_test in the tests directory. Also, it is unable to pass since we no longer return an array, and PrintAndParse does not control anything with the channel. Literally the only thing that this would be testing is that the line
```
strings.Replace(text, "\n", " ", -1)
```
is doing what it's supposed to be doing.

Tbh, probs should've deleted this method and just call getWords instead inside of retrieveCorrectTypes.

Reply:
Yeah let’s do that get rid of print and parse my bad